info.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "fortiSOARForMicrosoftTeams",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "",
+    "type": "solutionpack",
+    "label": "FortiSOAR For Microsoft Teams",
+    "dependencies": [
+        {
+            "name": "sOARFramework",
+            "type": "solutionpack",
+            "label": "SOAR Framework",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "FortiSOAR for Microsoft Teams application allows teams to leverage the power of FortiSOAR as part of their daily communications.",
```

Comment:
This needs to be reviewed and enhanced. Please get some inputs from Amit and Abhishek Shukla

Reply:
Will update this part once documentation done.